management-core/src/main/java/com/ns/task/service/ProductCoreServiceImpl.java

```diff
@@ -38,9 +40,15 @@ public ProductCoreServiceImpl(ModelMapper mapper, ProductRepository repository)
     @Override
     public ProductDTO insertProduct(ProductDTO product) {
         ProductEntity productEntity = convertToEntity(product);
-        logger.debug("Sending data to DB {}", productEntity);
-        ProductEntity productSaved = repository.saveProduct(productEntity);
-        logger.debug("Getting persisted data from insert to DB {}", productSaved);
+        ProductEntity productSaved = new ProductEntity();
+        try {
+            logger.debug("Sending data to DB {}", productEntity);
+            productSaved = repository.saveProduct(productEntity);
```

Comment:
is `productSaved` different to `productEntity ` ?

Reply:
You are right I was being redundant